### PR TITLE
chore: initialize changesets

### PR DESCRIPTION
**Pull request by with-heart**
[`changesets`](https://github.com/atlassian/changesets) won't save us from our lack of change documentation recently 🙃 but it can help with the future

A changeset can be added locally by running `yarn changeset`, but we can also manage them using [`changeset-bot`](https://github.com/apps/changeset-bot).

We'll be able to use [`changesets/action`](https://github.com/changesets/action) to automatically version and publish packages.

---

If you look at `.changeset/pretty-icons-rule.md` (a randomly-generated name), you can see the changeset for this set of changes. In order to make `changesets` work, I had to set the versions for `chakra-cra`, `chakra-nextjs`, and `@chakra-ui/docs` to `1.0.0-next.4`. When `changeset version` is run, this will automatically add the changeset message from that file to `CHANGELOG.md` in each package.

There's currently not a way to generate a "combined" changelog for the repo as a whole, but there's an issue open for this (see atlassian/changesets#387) so I'm going to investigate adding this functionality in preparation for our 1.0 release.

### CODE REVIEW

**Comment by with-heart** [comment]
I'm going to try to generate changesets for changes made since `1.0.0-next.4` so we can get the more important changes documented.

File: .changeset/config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.3.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "linked": [],
+  "access": "restricted",
```
Comment (neighborhood999) [inline .changeset/config.json:6]:
```suggestion
  "access": "public",
```

ref: [changelong](https://github.com/atlassian/changesets/blob/13406e99ef0927361b36c345f3fc51acf7885e40/packages/get-workspaces/CHANGELOG.md#patch-changes-4)

**Comment by segunadebayo** [comment]
Thank you for this @with-heart. I agree we need to improve our documentation and Changeset will help us with this, and even more.

Kindly review @neighborhood999 comment

**Comment by with-heart** [comment]
Yeap, we can definitely integrate with the action! Haven't added that yet but will soon.

@segunadebayo I also put in a request to add the changesets bot to this repo and the cli repo. Not sure if that shows a notification for you or how that works.

**Comment by with-heart** [comment]
I'm not normally one to merge my own PRs, but I want to try to use this for releasing `1.0.0-next.5`, so I'm gonna merge it. 🤞

**Pull request merged**